.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
```

Comment:
Do we really want to encourage _not_ using the issue templates?

Reply:
ahh sorry that was a copy paste error. Will remove